torchmetrics/image/ergas.py

```diff
@@ -0,0 +1,96 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict, List, Union
+
+from torch import Tensor
+from typing_extensions import Literal
+
+from torchmetrics.functional.image.ergas import _ergas_compute, _ergas_update
+from torchmetrics.metric import Metric
+from torchmetrics.utilities import rank_zero_warn
+from torchmetrics.utilities.data import dim_zero_cat
+
+
+class ErrorRelativeGlobalDimensionlessSynthesis(Metric):
+    """Relative dimensionless global error synthesis (ERGAS) is used to calculate the accuracy of Pan sharpened
+    image considering normalized average error of each band of the result image
+    (ErrorRelativeGlobalDimensionlessSynthesis).
+
+    Args:
+        ratio: ratio of high resolution to low resolution
+        reduction: a method to reduce metric score over labels.
+
+            - ``'elementwise_mean'``: takes the mean (default)
+            - ``'sum'``: takes the sum
+            - ``'none'``: no reduction will be applied
+
+        kwargs:
+            Additional keyword arguments, see :ref:`Metric kwargs` for more info.
+
+    Return:
+        Tensor with ErrorRelativeGlobalDimensionlessSynthesis score
+
+    Example:
+        >>> import torch
+        >>> from torchmetrics import ErrorRelativeGlobalDimensionlessSynthesis
+        >>> preds = torch.rand([16, 1, 16, 16], generator=torch.manual_seed(42))
+        >>> target = preds * 0.75
+        >>> ergas = ErrorRelativeGlobalDimensionlessSynthesis()
+        >>> ergas(preds, target)
+        tensor(153.9085)
+
+    References: Qian Du; Nicholas H. Younan; Roger King; Vijay P. Shah, "On the Performance Evaluation of
+    Pan-Sharpening Techniques" in IEEE Geoscience and Remote Sensing Letters, vol. 4, no. 4, pp. 518-522,
+    15 October 2007, doi: 10.1109/LGRS.2007.896328.
+    """
+
+    preds: List[Tensor]
+    target: List[Tensor]
+    higher_is_better: bool = False
+
```

Comment:
is the metric differentiable?
```suggestion
    higher_is_better: bool = False
    is_differentiable: bool = False

```

Reply:
yes, this metric is differentiable